kolibri/plugins/facility/assets/src/views/UserPage/index.vue

```diff
@@ -529,6 +545,37 @@
           },
         ];
       },
+      hasSelectedUsers() {
+        return this.selectedUsers && this.selectedUsers.size > 0;
+      },
+      listContainsLoggedInUser() {
+        return this.selectedUsers.has(this.currentUserId);
+      },
+      canAssignCoaches() {
+        if (!this.hasSelectedUsers) return false;
+        return this.facilityUsers
+          .filter(user => this.selectedUsers.has(user.id))
+          .every(
+            user =>
+              user.kind === UserKinds.COACH ||
+              user.kind === UserKinds.ADMIN ||
+              user.kind === UserKinds.SUPERUSER ||
+              user.is_superuser,
+          );
+      },
+      canEnrollOrRemoveFromClass() {
+        if (!this.hasSelectedUsers) return false;
+        return this.facilityUsers
+          .filter(user => this.selectedUsers.has(user.id))
+          .every(
+            user =>
+              user.kind === UserKinds.LEARNER ||
+              user.kind === UserKinds.COACH ||
```

Comment:
I'm seeing that the Facility coaches have the kind "coach" and class coaches have the kind "classroom assignable coach". This means that this check is not including class coaches and when I select a classroom coach, I can only delete that user. I had a similar problem when enrolling users in the side panel and had to [check if the user kind included UserKinds.COACH](https://github.com/learningequality/kolibri/blob/b31fbff1ca1033702d861ac95564abbb078c3ea2/kolibri/plugins/facility/assets/src/views/users/sidePanels/EnrollLearnersSidePanel.vue#L212) rather than being equal to it.

https://github.com/user-attachments/assets/de2000bd-1f03-4df7-81b1-deb519669f64